pkg/tfbridge/auto_aliasing.go

```diff
@@ -226,7 +236,7 @@ func aliasResource(
 	// Note: If the user explicitly sets a MaxItemOne value, that value is respected
 	// and overwrites the current history.'
 
-	if res == nil {
+	if res == nil || hist[tfToken] == nil {
```

Comment:
This change means we skip traversing all the fields of the resource which seems to account for ~60% of the speedup